meta/lvasgn.rb

```diff
@@ -6,3 +6,28 @@
   mutation 'a = false'
   mutation 'a = nil'
 end
+
+Mutant::Meta::Example.add :lvasgn do
+  source 'a, b = foo'
+
+  singleton_mutations
+  mutation 'a__mutant__, b = foo'
+  mutation 'a, b__mutant__ = foo'
+  mutation 'b, = foo'
+  mutation 'a, = foo'
+end
+
+Mutant::Meta::Example.add :lvasgn do
+  source 'a = nil; a'
+
+  mutation 'a = nil; nil'
+  mutation 'a = nil; self'
+  mutation 'a = nil'
+  # TODO: fix invalid AST
+  #   These ASTs are not valid and should NOT be emitted
+  #   Mutations of lvarasgn need to be special cased to avoid this.
```

Comment:
Is the issue here that we shouldn't replace an assignment with a noop in general? Or is it a question of not eliminating the assignment when we can statically check if the lvar is used below?